### PR TITLE
Use pip-installed supervisor

### PR DESCRIPTION
**Pull request by acrellin**

### CODE REVIEW

**Comment by stefanv** [comment]
Only the one small change we discussed this morning, then I think we're good to go.

**Comment by acrellin** [comment]
That wasn't clear - can you please explain (or provide an example in a
comment)?

**Comment by acrellin** [comment]
(It's unclear how a format string would add clarity)

File: tools/supervisor_status.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 def supervisor_status():
     result = subprocess.run(
-        'supervisorctl -c baselayer/conf/supervisor/supervisor.conf status',
+        'python -m supervisor.supervisorctl -c baselayer/conf/supervisor/supervisor.conf status',
         shell=True, cwd=base_dir, check=True,
```
Comment (stefanv) [inline tools/supervisor_status.py]: Maybe something like:

```
# top of the file
config = 'baselayer/conf/supervisor/supervisor.conf'
supervisorctl = ['python', '-m', 'supervisor.supervisorctl', '-c', config]

# further down
result = subprocess.run(supervisorctl + ['status'], ...)
```

**Comment by acrellin** [comment]
Even though we never run this command with any other options? This extra
verbosity makes it less clear to me, actually.

**Comment by acrellin** [comment]
If it were something we were doing multiple times with different options in
this file, then I could see the case for that.

**Pull request merged**